package.json

```diff
@@ -1,8 +1,16 @@
 {
   "name": "athapt",
   "license": "MIT",
-  "private": true,
-  "workspaces": [
-    "packages/*"
-  ]
+
+  "scripts": {
+    "lint": "eslint src/**/*.js"
+  },
+
+  "devDependencies": {
+    "eslint": "^7.10.0",
+    "eslint-config-google": "^0.14.0"
```

Comment:
Why do we need `eslint-config-google`

Reply:
its a style guide. A style guide is a set of standards that outline how code should be written and organized. A style guide contains general rules about “how to write” code, e.g. which quotes to use, how many spaces to indent, where to put line breaks, etc.

it is said: “All code in any code-base should look like a single person typed it, no matter how many people contributed.”